app-pda/ideviceinstaller/ideviceinstaller-1.1.1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A tool to interact with the installation_proxy of an Apple's iDevice"
+HOMEPAGE="http://www.libimobiledevice.org/"
+SRC_URI="https://github.com/libimobiledevice/ideviceinstaller/releases/download/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

Comment:
also, since it's a new version you can't directly commit stable packages.
```
KEYWORDS="~amd64 ~x86"
```

Reply:
I should push forced all those fixes. Let me try it. Thanks for reviewing.